From 3b38eb317bc9f6ed409accb174d1fb52af8aea8b Mon Sep 17 00:00:00 2001
From: nulano <nulano@nulano.eu>
Date: Sun, 30 Aug 2020 00:57:27 +0200
Subject: [PATCH] remove legacy include directory for libimagequant

1) it is hardcoded to 32-bit C:\msys64\mingw32
2) current library version uses \mingw64\include\libimagequant.h directly
---
 setup.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/setup.py b/setup.py
index cf39162502..528a3a0448 100755
--- a/setup.py
+++ b/setup.py
@@ -528,11 +528,6 @@ def build_extensions(self):
             _add_directory(library_dirs, "/lib")
 
         if sys.platform == "win32":
-            if PLATFORM_MINGW:
-                _add_directory(
-                    include_dirs, "C:\\msys64\\mingw32\\include\\libimagequant"
-                )
-
             # on Windows, look for the OpenJPEG libraries in the location that
             # the official installer puts them
             program_files = os.environ.get("ProgramFiles", "")
